app/code/Magento/Contact/Api/ConfigInterface.php

```diff
@@ -0,0 +1,59 @@
+<?php
+/**
+ * Contact module base controller
+ */
+namespace Magento\Contact\Api;
+
+/**
+ * Contact module configuration
+ */
+interface ConfigInterface
```

Comment:
Thank you for the review. I did not intend to expose the interfaces to the web api, they were only meant as service contracts. I was not aware of the convention to distinguish between `Api` namespace and `@api`  annotation, will change it as suggested